Feature/play with computer

hanzzb opened the pull request:
컴퓨터와 플레이하는 기능을 함수로 만들어 추가했습니다.

seohyeonKK committed:
ADD:Add the feature that play with computer

Add the feature that play with computer as a comment.

hanzzb committed:
ADD:Add the feature play with computer in code

Add comments' feature to code.

seohyeonKK commented:
merge go

cherry2030 commented:
gogogogo~~~

mjseok commented on Code.c:
```diff
@@ -391,3 +399,181 @@ void chooseMenu()
+{
+printf("Type any digit from 1-9 to fill your response:- ");
+scanf("%s", board_position);
+if (strcmp(board_position, "1") == 0 && board_symbol[0] == '1')
+board_symbol[0] = symbol;
+else if (strcmp(board_position, "2") == 0 && board_symbol[1] == '2')
+board_symbol[1] = symbol;
+else if (strcmp(board_position, "3") == 0 && board_symbol[2] == '3')
+board_symbol[2] = symbol;
+else if (strcmp(board_position, "4") == 0 && board_symbol[3] == '4')
+board_symbol[3] = symbol;
+else if (strcmp(board_position, "5") == 0 && board_symbol[4] == '5')
+board_symbol[4] = symbol;
+else if (strcmp(board_position, "6") == 0 && board_symbol[5] == '6')
+board_symbol[5] = symbol;
+else if (strcmp(board_position, "7") == 0 && board_symbol[6] == '7')
+board_symbol[6] = symbol;
+else if (strcmp(board_position, "8") == 0 && board_symbol[7] == '8')
+board_symbol[7] = symbol;
+else if (strcmp(board_position, "9") == 0 && board_symbol[8] == '9')
+board_symbol[8] = symbol;
+else
```
함수화해서 올리는게 더 좋을 것 같아요!

hanzzb committed:
ENH:Functionalize computer feature

Implement computer features with existing functions.
Make checkposition2() to check player's turn.

mjseok commented:
merge합시당~

seohyeonKK commented:
so goooooooood 💯

cherry2030 commented:
merge 합싲당~~~~~@@~!~
